.gitignore

```diff
@@ -114,5 +114,3 @@ ENV/
 *.code-workspace
 
 .DS_Store
-
-pygeoapi/django_pygeoapi/sample_project/db.sqlite3
```

Comment:
Is it now `examples/django/sample_project/db.sqlite3` to be ignored?

Reply:
Done